Finjector.Web/ClientApp/src/pages/Details.test.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+
+import { MemoryRouter, Route, Routes } from "react-router-dom";
+import { render, screen, waitFor } from "@testing-library/react";
+import { QueryClientProvider, QueryClient } from "@tanstack/react-query";
+import { describe, it, expect } from "vitest";
+import { server } from "../../test/mocks/node";
+import {
+  fakeCharts,
+  fakeInvalidChart,
+  fakeValidAeDetails,
+  fakeValidChart,
+} from "../../test/mocks/mockData";
+import Details from "./Details";
+
+beforeAll(() => server.listen());
+beforeEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+const testChart = fakeCharts[0];
```

Comment:
This is the same as the fakeValidChart imported?

Reply:
@jSylvestre yea, same.  in retrospect probably should have just used that one, but it doesn't matter in practice